File: module06/module257/module2/src/main/java/com/android/example/module06_module257_module2/ClassAAD.java
package com.android.example.module06_module257_module2;

public class ClassAAD {
	private 	com.android.example.module06_module257_module1.ClassAAB instance_var_1_0 = new com.android.example.module06_module257_module1.ClassAAB();
	private 	com.android.example.module06_module257_module3.ClassAAF instance_var_1_1 = new com.android.example.module06_module257_module3.ClassAAF();
	private 	com.android.example.module06_module257_module1.ClassAAE instance_var_1_2 = new com.android.example.module06_module257_module1.ClassAAE();
	private 	com.android.example.module06_module257_module3.ClassAAB instance_var_1_3 = new com.android.example.module06_module257_module3.ClassAAB();

	public void method0(
		com.android.example.module06_module257_module3.ClassAAA param0,
		com.android.example.module06_module257_module3.ClassAAH param1,
		com.android.example.module06_module257_module3.ClassAAC param2) throws Throwable {
		java.util.Collections.emptyList().forEach( lambda0 -> {
			try {
				for (int iAb = 0; iAb < 5; iAb++) {
					io.reactivex.subscribers.TestSubscriber<Object> local_var_4_0 = new io.reactivex.subscribers.TestSubscriber();
					local_var_4_0.hasSubscription();

					io.reactivex.internal.schedulers.SingleScheduler local_var_4_1 = new io.reactivex.internal.schedulers.SingleScheduler();
					local_var_4_1.shutdown();

					io.reactivex.subscribers.TestSubscriber<Object> local_var_4_2 = new io.reactivex.subscribers.TestSubscriber();
					local_var_4_2.onNext(new java.lang.Object());

					local_var_4_0.dispose();

				}
				io.reactivex.subscribers.TestSubscriber<Object> local_var_3_0 = new io.reactivex.subscribers.TestSubscriber();
				local_var_3_0.onNext(new java.lang.Object());

			} catch(Throwable e) { } // ignore
		});
		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_3 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_3.request(-840993338l);

		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_4 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_4.request(-786345230l);

	}

	public void method1(
		com.android.example.module06_module257_module3.ClassAAH param0,
		com.android.example.module06_module257_module1.ClassAAB param1,
		com.android.example.module06_module257_module3.ClassAAF param2,
		com.android.example.module06_module257_module1.ClassAAG param3) throws Throwable {
	}

	public void method2(
		com.android.example.module06_module257_module1.ClassAAA param0) throws Throwable {
		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_1 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_1.onNext(new java.lang.Object());

		io.reactivex.internal.disposables.ListCompositeDisposable local_var_2_2 = new io.reactivex.internal.disposables.ListCompositeDisposable();
		local_var_2_2.clear();

	}

	public void method3(
		com.android.example.module06_module257_module3.ClassAAB param0,
		com.android.example.module06_module257_module3.ClassAAB param1,
		com.android.example.module06_module257_module3.ClassAAD param2,
		com.android.example.module06_module257_module1.ClassAAE param3) throws Throwable {
		param0.method0(new io.reactivex.subscribers.TestSubscriber());

		param0.method0(new io.reactivex.subscribers.TestSubscriber());

	}

	public void method4(
		com.android.example.module06_module257_module3.ClassAAF param0) throws Throwable {
		io.reactivex.internal.schedulers.IoScheduler local_var_2_1 = new io.reactivex.internal.schedulers.IoScheduler();
		local_var_2_1.createWorker();

		param0.method0(new io.reactivex.subscribers.TestSubscriber(), new io.reactivex.subscribers.TestSubscriber(), new io.reactivex.subscribers.TestSubscriber());

		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_2 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_2.assertSubscribed();

	}
}

File: module06/module333/module2/src/main/java/com/android/example/module06_module333_module2/ClassAAD.kt
package com.android.example.module06_module333_module2

@Suppress("UNUSED_PARAMETER")
class ClassAAD {
	private 	val instance_var_1_0: com.android.example.module06_module333_module5.ClassAAG = com.android.example.module06_module333_module5.ClassAAG()
	private 	val instance_var_1_1: com.android.example.module06_module333_module5.ClassAAA = com.android.example.module06_module333_module5.ClassAAA()
	private 	val instance_var_1_2: com.android.example.module06_module333_module5.ClassAAF = com.android.example.module06_module333_module5.ClassAAF()
	private 	val instance_var_1_3: com.android.example.module06_module333_module5.ClassAAJ = com.android.example.module06_module333_module5.ClassAAJ()

	fun method0(
		param0: com.android.example.module06_module333_module5.ClassAAD,
		param1: com.android.example.module06_module333_module5.ClassAAC,
		param2: com.android.example.module06_module333_module5.ClassAAH
	) {
	}

	fun method1(
		param0: com.android.example.module06_module333_module5.ClassAAJ
	) {
		param0.method1(io.reactivex.subscribers.TestSubscriber(), io.reactivex.subscribers.TestSubscriber(), io.reactivex.internal.schedulers.SingleScheduler())

		val local_var_2_1: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_1.assertNotSubscribed()

		param0.method0(io.reactivex.internal.disposables.ListCompositeDisposable(), io.reactivex.internal.disposables.ListCompositeDisposable(), io.reactivex.subscribers.TestSubscriber())

		val local_var_2_2: io.reactivex.internal.disposables.ListCompositeDisposable = io.reactivex.internal.disposables.ListCompositeDisposable()
		local_var_2_2.clear()

	}

	fun method2(
		param0: com.android.example.module06_module333_module5.ClassAAG,
		param1: com.android.example.module06_module333_module5.ClassAAE
	) {
		param0.method1(io.reactivex.subscribers.TestSubscriber(), io.reactivex.internal.disposables.ListCompositeDisposable(), io.reactivex.subscribers.TestSubscriber(), io.reactivex.subscribers.TestSubscriber())

		val local_var_2_2: io.reactivex.internal.disposables.ListCompositeDisposable = io.reactivex.internal.disposables.ListCompositeDisposable()
		local_var_2_2.dispose()

	}

	fun method3(
		param0: com.android.example.module06_module333_module5.ClassAAE,
		param1: com.android.example.module06_module333_module5.ClassAAB
	) {
		val local_var_2_2: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_2.dispose()

		param0.method1(io.reactivex.subscribers.TestSubscriber(), io.reactivex.disposables.SerialDisposable(), io.reactivex.subscribers.TestSubscriber())

	}
}
